.github/workflows/tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Run CI tests
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  shellcheck:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run shellcheck
+        run: |
+          sudo apt-get install shellcheck
```

Comment:
This may need an `sudo apt update &&` to mitigate against repo updates

Reply:
You're right, I just copied this over from the travis code without thinking about it.

Reply:
I think there's a good chance that shellcheck is included natively on the Ubuntu image

Reply:
@jasonkarns It might be, but using update and install should ensure we get the latest available version. When I started with `ubuntu-latest` we got 16.04 which gave us shellcheck 0.4...